Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,29 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+order.forEach(({ itemName, quantity, unitPricePence }) => {
+  let totalPence = quantity * unitPricePence;
+  let pence = totalPence % 100;
+  let paddedPence = String(pence).padStart(2, "0");
+  let pounds = Math.floor(totalPence / 100);
+  let priceEachItem = `${pounds}.${paddedPence}`;
+
+  console.log(
+    `${String(quantity).padEnd(7, " ")}${itemName.padEnd(
+      20,
+      " "
+    )}${priceEachItem}`
+  );
+});
+let sumAllPence = 0;
+order.forEach(({ quantity, unitPricePence }) => {
```

Comment:
Suggestion: you could write this with reduce and it would be much cleaner, this is a perfect use case for reduce.